oracles/README.md

```diff
@@ -1,26 +1,57 @@
-# Local oracle setup
+# Galadriel Oracle
 
+## Prerequisites
 
-## Setup
+### Dependencies 
+- Docker and Docker Compose (for Docker setup only)
+- Python 3.11+ (for Manual setup)
+- [A deployed smart contract address](https://github.com/galadriel-ai/contracts/blob/main/contracts/README.md)
+- [Serper API key](https://serper.dev) - required by `web_search` tool
+- [OpenAI API key](https://openai.com) - LLM inference
 
```

Comment:
I would just add - Private key as a separate requirement here not as the next sentence maybe idk